database/sql/src/main/scala/cromwell/database/slick/tables/CallCachingAggregationEntryComponent.scala

```diff
@@ -50,6 +50,20 @@ trait CallCachingAggregationEntryComponent {
     } yield ()).exists
   )
 
+  val existsCallCachingEntriesForBaseAggregationHashWithExecutionBucket = Compiled(
```

Comment:
ToL: the "buckets" in the name and variable names might be overly specific as this really just checks that the detritus starts with `executionBucket` but it could be anything